templates/base.html

```diff
@@ -21,9 +21,17 @@
         <meta name="author" content="{{ AUTHOR }}" />
         {% endif %}
 
+        <!-- Fill in Site ICON-->
```

Comment:
When you put in the base, it will conflict with the feature image of the article.

It should be placed in index.html unless I am missing something.

Reply:
Ok, let me do it

Reply:
In this case, to make it cleaner I would switch to 'base.html' the code so that article uses local one if artciel and use global one if not

Reply:
Moved from the macro that only shows on articles in smo_metadata  to base.html having the if there